VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -132,7 +132,9 @@ def disk_list_for_billing(self):
                 self.logger.log("LUN Number {0}, disk {1}".format(lunNumber,device_name))   
             self.logger.log("Disks to be included {0}".format(self.disksToBeIncluded))
         else:
-            self.logger.log("There is some glitch in executing the command sudo lsscsi, So the lsscsi list is empty and the Billing will not be done ")
+            if(self.isAnyDiskExcluded == True):
+                self.size_calc_failed = True
+                self.logger.log("There is some glitch in executing the command sudo lsscsi, So the lsscsi list is empty and the Billing will not be done ")
```

Comment:
This message should be logged in else block of if(len(..) != 0)) instead of under if (self.isAnyDiskExcluded == True)?

Reply:
Actually this will only occur when the disks have been excluded and lsscsi got failed, so removed this check here and added at the time we call this function.